csrankings-y.csv

```diff
@@ -620,7 +620,7 @@ Yuan Hong,Illinois Institute of Technology,https://science.iit.edu/people/facult
 Yuan Jiang 0001,Nanjing University,http://lamda.nju.edu.cn/jiangy,Qp_v9woAAAAJ
 Yuan Luo 0003,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/en/PeopleDetail.aspx?id=155,NOSCHOLARPAGE
 Yuan Tang,Fudan University,http://people.csail.mit.edu/yuantang,kC2YD14AAAAJ
-Yuan Tian 0001,University of Virginia,https://www.ytian.info,ja0GtqgAAAAJ
+Yuan Tian 0001,Univ. of California - Los Angeles,https://www.ytian.info,ja0GtqgAAAAJ

```

Comment:
Please clarify that you can solely advise PhD students *in the CS department*. Thanks.

Reply:
Closing. Please respond if you are eligible for inclusion, and point to evidence - thanks.